ukrainian/wait-more/wait.md

```diff
@@ -6,3 +6,4 @@
 Що будете робити далі?
 
 [Звичайно ж вийду з хатинки!](out/index.md)
+[Йти в дорогу не поснідавши? Це вже ні. Краще скуштую стіну з зефіру..](eat-wall/obese/diabetes.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.